tools/abel/README.md

```diff
@@ -0,0 +1,75 @@
+# ABEL
+#### Automated Building Entity Loader
+
+## Overview
+ABEL is a tool allowing system integrators to easily and efficiently create and modify [Building Configuration files](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md).
+
+#### UDMI
```

Comment:
I think it is better to comment this section for now since it is not yet out

Reply:
Ok, Done.